frontend/pages/software/components/EmptySoftwareTable.tsx

```diff
@@ -0,0 +1,68 @@
+// This component is used on DashboardPage.tsx > Software.tsx, Host Details/Device User > Software.tsx, and ManageSoftwarePage.tsx
+
+import React from "react";
+
+import CustomLink from "components/CustomLink";
+import EmptyTable from "components/EmptyTable";
+import { IEmptyTableProps } from "interfaces/empty_table";
+
+export interface IEmptySoftwareTableProps {
+  isSoftwareDisabled?: boolean;
+  isFilterVulnerable?: boolean;
+  isSandboxMode?: boolean;
+  isCollectingSoftware?: boolean;
+  isSearching?: boolean;
+}
+
+const EmptySoftwareTable = ({
+  isSoftwareDisabled,
+  isFilterVulnerable,
+  isSandboxMode,
+  isCollectingSoftware,
+  isSearching,
+}: IEmptySoftwareTableProps): JSX.Element => {
+  const emptySoftware: IEmptyTableProps = {
+    header: `No ${
+      isFilterVulnerable ? "vulnerable " : ""
+    }software match the current search criteria`,
+    info: `Try again in about ${
+      isSandboxMode ? "5 minutes" : "1 hour"
+    } as the system catches up.`,
+  };
+  if (isCollectingSoftware) {
+    emptySoftware.header = "No software detected";
+    emptySoftware.info = `This report is updated every ${
+      isSandboxMode ? "5 minutes" : "hour"
+    } to protect the performance of your devices.`;
```

Comment:
@zayhanlon 

No software detected

Fleet begins collecting software inventory after a host is enrolled.